src/frontend/src/handler/describe.rs

```diff
@@ -99,33 +100,34 @@ pub fn handle_describe(handler_args: HandlerArgs, table_name: ObjectName) -> Res
     // Convert all column descs to rows
     let mut rows = col_descs_to_rows(columns);
 
+    fn fmt_col_project<T, C>(columns: &[ColumnDesc], cb: C) -> String
+    where
+        T: Display + ?Sized,
+        C: FnMut(&ColumnDesc) -> &T,
+    {
+        format!(
+            "{}",
+            display_comma_separated(&columns.iter().map(cb).collect::<Vec<_>>())
+        )
+    }
```

Comment:
Because a previous commit mapped (projected) vec of struct to vec of field (`description`)... (`fmt_col_project(&columns, |x| &x.description)`).
Now only `x.name` is mapped, and unnecessary parameters can indeed be deleted.